document-generation/smartdocuments/src/test/kotlin/com/ritense/smartdocuments/client/SmartDocumentsClientTest.kt

```diff
@@ -215,13 +212,7 @@ internal class SmartDocumentsClientTest : BaseTest() {
                 )
             )
         }
-
-        assertEquals(
-            "400 The server cannot or will not process the request due to something that is perceived to be a client " +
-                    "error (e.g., no valid template specified, user has no privileges for the template, malformed request syntax, " +
-                    "invalid request message framing, or deceptive request routing). Response received from server:\n$responseBody",
-            exception.message?.trimIndent()
-        )
+        assertThat(exception.statusCode).isEqualTo(HttpStatus.BAD_REQUEST)
```

Comment:
Maybe also check the exception.message like was done previously

Reply:
This is pain in the ass to get right and really doesn't add any value imo.